js/src/LinearScaleModel.js

```diff
@@ -47,7 +47,7 @@ var LinearScaleModel = scalemodel.ScaleModel.extend({
 
     reverse_changed: function() {
         this.reverse = this.get("reverse");
-        if(this.domain.length > 0 && this.reverse) {
+        if(this.domain.length > 0) {
```

Comment:
But it reverses with respect to the previous state on change of the attribute reverse, which is fine. Also, with the previous change, if reverse is set to `False` from `True` nothing happens. If you can give me the code to reproduce the bug, it will be great.